### PR TITLE
Add Documentation link in CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by apsknight**
As suggested in https://github.com/jupyterlab/jupyterlab/issues/3614, I have added documentation link in CONTRIBUTING.md

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -271,3 +271,5 @@ https://jupyterlab-tutorial.readthedocs.io/en/latest/index.html.
 
 - The npm modules are fully compatible with Node/Babel/ES6/ES5. Simply
 omit the type declarations when using a language other than TypeScript.
+
+- For more information, read the [documentation](http://jupyterlab-tutorial.readthedocs.io/en/latest/).
```
Comment (blink1073) [inline CONTRIBUTING.md]: Thanks!  Our documentation has moved to `http://jupyterlab.readthedocs.io/en/latest/`

Comment (jzf2101) [reply CONTRIBUTING.md]: Also I was explicitly referring to http://jupyterlab.readthedocs.io/en/stable/developer/documentation.html

Comment (jzf2101) [reply CONTRIBUTING.md]: Actually I meant http://jupyterlab.readthedocs.io/en/stable/developer/documentation.html

**Comment by blink1073** [comment]
Thanks!

**Pull request merged**